apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/MetricsSummary/Charts/Numerical/CostOverResults.tsx

```diff
@@ -75,8 +75,12 @@ export function CostOverResultsChart({
             xAxis: {
               label: 'Average result',
               type: 'number',
-              min: evaluation.configuration.detail!.range.from,
-              max: evaluation.configuration.detail!.range.to,
+              min: (
+                evaluation.configuration ?? evaluation.metadata.configuration
```

Comment:
This is all over the place. Do a utils functions to get this data

Reply:
This check is just for the transition PR, so that prod does not break between migrations. It will be removed in the next PR, so I don't think it is needed